src/static.cpp

```diff
@@ -110,10 +110,10 @@ void DynamicNumber<int32_t>::updateText()
     const char* p = prefix ? prefix : "";
     const char* s = suffix ? suffix : "";
     if ((textFlags & PREC2) == PREC2) {
-      lv_label_set_text_fmt(lvobj, "%s%" PRId32 ".%02" PRId32 "%s", p,
+      lv_label_set_text_fmt(lvobj, "%s%" PRId32 ".%02" PRIu32 "%s", p,
                             value / 100, value % 100, s);
```

Comment:
Shouldn't we use something like `abs()` to get a positive value?

Reply:
Of course! :-)